x-pack/plugins/secops/public/components/timeline/body/column_headers/text_filter/text_filter.test.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount } from 'enzyme';
+import * as React from 'react';
+import { DEFAULT_PLACEHOLDER, TextFilter } from './text_filter';
+
+describe('TextFilter', () => {
+  describe('rendering', () => {
+    describe('placeholder', () => {
+      test('it renders the default placeholder when no filter is specified, and a placeholder is NOT provided', () => {
+        const wrapper = mount(<TextFilter columnId="foo" minWidth={100} />);
+
+        expect(wrapper.find(`input[placeholder="${DEFAULT_PLACEHOLDER}"]`)).toHaveLength(1);
```

Comment:
Yea, seems like exists in these cases would be better than checking lengths
https://github.com/airbnb/enzyme/blob/master/docs/api/ReactWrapper/exists.md